app.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+require 'sinatra/flash'
+require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative './lib/peep'
+require_relative './lib/user'
+require_relative './spec/database_connection_setup'
+require_relative './spec/login_helper'
+
+class Chitter < Sinatra::Base
+  enable :method_override
+  enable :sessions
+
+  register Sinatra::Flash
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    if logged_in?
+      redirect '/chitter'
+    else
+      erb :prompt
+    end
+  end
+
+  post '/signup' do
```

Comment:
Your controller is quite fat, try to refactor any logic that isn't about the response to the model where possible